.travis.yml

```diff
@@ -19,7 +19,7 @@ language: python
 # Available Python versions:
 # http://about.travis-ci.org/docs/user/ci-environment/#Python-VM-images
 python:
-  - "2.7"
+  - "3.7-dev"
```

Comment:
On trusty, 3.7-dev points to an ancient commit before 3.7.0, as [OpenSSL on trusty is too old](https://github.com/travis-ci/travis-ci/issues/9069):

> Python 3.7.0a4+

(https://travis-ci.org/buildbot/buildbot/jobs/474355722#L562)

I guess you want the latest Python 3.7? Then you need to change `dist: trusty` to `dist: xenial`

Reply:
OK, I switched it to Python 3.7.1 on xenial